Jenkinsfile

```diff
@@ -137,6 +137,9 @@ pipeline {
                     script {
                         env.TAG = sh(script: "grep Version DESCRIPTION | head -n1 | cut -d':' -f2", returnStdout: true).trim()
                     }
+
+                    // this solves git's "dubious ownership" complaint, but we have no idea why and how only this repo is suddenly affected
+                    sh "git config --global --add safe.directory /home/jenkins/agent/workspace/enis_molgenis-r-armadillo_master"
```

Comment:
Does the directory not have to be: /home/jenkins/agent/workspace/molgenis_molgenis-r-armadillo_master?